main.c

```diff
@@ -469,6 +469,22 @@ Bool is_input_ev(Display *dpy, XEvent *ev, XPointer arg)
 	return ev->type == ButtonPress || ev->type == KeyPress;
 }
 
+void init_key_handler(void)
+{
+	extprefix = true;
+	close_info();
+	snprintf(win.bar.l.buf, win.bar.l.size, "Getting key handler input "
+	         "(%s to abort)...", XKeysymToString(keyhandler_abort));
+	win_draw(&win);
```

Comment:
We don't need to draw the window here. I beleive we'll redraw if we return true to `cg_prefix_external`

Reply:
>  I beleive we'll redraw if we return true to cg_prefix_external

Yes, but that will cause a full redraw.